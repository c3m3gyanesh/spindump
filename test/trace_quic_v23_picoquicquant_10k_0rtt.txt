Quant client vs. Picoquic server, using 0rtt connection.
