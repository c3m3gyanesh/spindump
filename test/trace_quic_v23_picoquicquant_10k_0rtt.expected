[
{ "Event": "new", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "0483f428-7433798045a578d470 (64751:4433)", "Ts": 1571088424815036, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "0483f428-0818c2c12c15a8af (64751:4433)", "Ts": 1571088424893073, "State": "Up", "Right_rtt": 78037, "Packets1": 1, "Packets2": 0, "Bytes1": 1228, "Bytes2": 0, "Bandwidth1": 1228, "Bandwidth2": 0 },
{ "Event": "spinflip", "Type": "QUIC", "Addrs": ["172.16.95.160","34.238.92.221"], "Session": "83f428a4-0818c2c12c15a8af (64751:4433)", "Ts": 1571088424980305, "State": "Up", "Transition": "0-1", "Who": "responder", "Packets1": 4, "Packets2": 12, "Bytes1": 1560, "Bytes2": 12822, "Bandwidth1": 1560, "Bandwidth2": 12822 }
]
